default.json

```diff
@@ -80,6 +80,17 @@
       "depNameTemplate": "mikefarah/yq",
       "datasourceTemplate": "github-releases"
     },
+    {
+      "fileMatch": [
+        "(^|/|\\.)Dockerfile$",
+        "(^|/)Dockerfile[^/]*$"
```

Comment:
Let's expand this to Makefiles as well please:
- https://github.com/rancher/elemental-operator/blob/80eb8469e31cb0b88923a0a2e26713e88928e4dc/Makefile#L48
- https://github.com/rancher-sandbox/cluster-api-provider-rke2/blob/e05b03eb7a3cc664ae2f75a11741d7b7cb0a3d53/Makefile#L73